site/content/docs/main/customize-installation.md

```diff
@@ -100,6 +100,10 @@ At installation, Velero sets default resource requests and limits for the Velero
 |Memory limit|512Mi|1024Mi|
 {{< /table >}}
 
+Depending on the cluster resources you, especially if you are using Restic, you may need to increase these limits. Through testing, the Velero maintainers have found these defaults to work well when backing up and restoring 1000 or less resources and total size of files per backup or restore is 100GB or below. If the resources you are planning to backup or restore exceed this, you will need to increase the CPU or memory resources available to Velero. In general, the Velero maintainer's testing found that backup operations needed more CPU & memory resources but were less time-consuming than restore operations, when comparing backing up and restoring the same amount of data. The exact CPU and memory limits you will need depend on the scale of the files and directories of your resources and your hardware. It's recommended that you perform your own testing to find the best resource limits for your clusters and resources. 
```

Comment:
'Depending on the cluster resources you, '
Looks like need to reword this sentence.

Reply:
thank you for reviewing, i've fixed this typo